### DISCUSSION TITLE
Code-Plugin for Unity Game Engine

### DISCUSSION THREAD

**Discussion by intelliot** [XRPLBounties/Proposals#15] [Closed (claimed or cancelled bounty)]
### Discussion thread for bounty: [Code-Plugin for Unity Game Engine](https://github.com/XRPLBounties/Proposals/blob/main/bounties/0015%20Code-Plugin%20for%20Unity%20Game%20Engine.md) ([PR](https://github.com/XRPLBounties/Proposals/pull/22))

- Enable developers to add XRPL functionality to their game
- Enable players to use XRPL functionality in games created with Unity

**Comment by dangell7** [comment]
Hello XRPL Bounties,

Unity has rejected the sdk. I figured they would. 

`Your package has features that are not compatible with the Standard Asset Store EULA (for example, it may have monetization, artificial limits, registration requirements, blockchain solutions, and similar). To have this asset package evaluated, please register as an Asset Store Service here: [link] (https://unity3d.com/asset-store/sell-assets/submission-guidelines , Section 1.5.b)`

1.5.b If you are an online service such as a monetization service, ad-network, back-end hosting service, analytics system, or other service where a variable amount of money changes hands after the user downloads your SDK or plugin, [register your product](https://create.unity3d.com/contact-vsp) as an Asset Store Service for it to be distributed on the Unity Asset Store.

I have registered my business but if the XRPLF wants to manage this, they would need to register as well.

**Reply by JST5000** [reply]
Ah, that's good to know - I didn't realize that was in the rules. Does that mean individual game builders each need to register with Unity in order to integrate with the XRPL and be offered on a Unity marketplace, or does this mean once you register with Unity that your plug in will be able to be offered to any game devs?

**Reply by dangell7** [reply]
It should be that I register and then anyone can use the plugin. I've been waiting for a rejection email but still hasen't come so I resubmitted the sdk. I will share the email with you when it comes. 

I also named it `Xrpl SDK for Unity` and I think they don't like that. Any ideas on a name change?

`Xrpl SDK`?

**Reply by JST5000** [reply]
XRPL SDK sounds good (I'd keep XRPL capitalized I think) - Alternatively XRPL Toolkit could also work

**Comment by dangell7** [comment]
<img width="1233" alt="Screen Shot 2022-09-25 at 2 29 28 PM" src="https://user-images.githubusercontent.com/25502424/192159473-15c787c5-6fb1-4d38-ba2d-2d3721bebf7d.png">

**Comment by jonathanlei** [comment]
Closing this bounty given the sdk rejection, thanks @dangell7 for submitting the work. Please let us know if there are updates as we are open to reopen this bounty if Unity policy changes.
